dm/dm-webui-guide.md

```diff
@@ -16,7 +16,6 @@ DM WebUI 是一个 TiDB Data Migration (DM) 迁移任务管理界面，方便用
 
 DM WebUI 主要包含以下界面：
 
-- **Dashboard**：包含了 DM 中同步任务的主要监控信息和运行状态信息，帮助用户快速了解任务的整体运行状况，以及与延迟、性能相关的重要信息。
```

Comment:
Maybe the picture `/media/dm/dm-webui-preview-cn.png` needs to remove `Dashboard` too.